### PR TITLE
Support Kafka 0.9.0.1

### PR DESCRIPTION
**Pull request by soid**
Kafka 0.9 dependency + API changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 Allows you to start and stop a Kafka broker + ZooKeeper instance for unit testing applications that communicate with Kafka.
 
-It uses Kafka version ```0.8.2.2``` and ZooKeeper version ```3.4.6```
+## Versions
+| kafka-unit | Kafka broker | Zookeeper |
+|------------|--------------|-----------|
+| 0.1.4      | 0.9.0.1      | 3.4.6     |
```
Comment (chbatey) [inline README.md]: lets make this a large change version e.g 0.2.0

**Comment by chbatey** [comment]
LGTM apart from the version in the read me. If you change that i'll release a new version to maven central

**Comment by soid** [comment]
Good point. Turns out there's 0.3 already http://mvnrepository.com/artifact/info.batey.kafka/kafka-unit
So I documented it as 0.4

**Comment by soid** [comment]
Thanks! Can you please publish the artifacts?

**Comment by chbatey** [comment]
Uploaded, normally visible in about 20 minuts.

**Pull request merged**